packages/cxl-ui/scss/global/cxl-accordion-card.scss

```diff
@@ -0,0 +1,76 @@
+@use "../mixins";
```

Comment:
Why is a new cxl-accordion-card styling model introduced here with this new file?

https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss is the current system, where a new theme should be defined.

See also below about `packages/storybook/cxl-ui/cxl-vaadin-accordion.stories.js` story name and model issues, probably related.

Reply:
You're breaking cxl-accordion-card appearance at the moment:

![image](https://user-images.githubusercontent.com/147228/134505862-1fb80d1c-82e0-42da-bc08-3dccc1410825.png)

cxl-accordion-card as currently defined, should stay 100% untouched.

You only add / overrides in a separate, or combined theme - an element can have multiple themes, something like `<cxl-vaadin-accordion theme="cxl-accordion-card cxl-accordion-playbook-card">`, and your story name would be `[theme="cxl-accordion-playbook-card"]` for short (fitting full theme info would be problematic).

Reply:
> You're breaking cxl-accordion-card appearance at the moment


@lkraav I thought there's gonna be **one** card standard in the new story, and old card themes like that are gonna be dropped/redesigned, due to the font inconsistency and other multiplying differences.
However we cannot skip the injection of the CSS to the component's Shadow Dom (internal parts have different font in the new theme). 
Should this be made in the theme's story file like this, or you have a better way?
```
registerStyles('cxl-accordion-card', css``);
```

Reply:
No one said anything about dropping existing themes.

We just need to separate playbook related accordion cards from course cards. Too different now.